src/pages/dashboard/Dashboard.tsx

```diff
@@ -101,11 +101,17 @@ const Dashboard = () => {
       [products, party]
     ) ?? [];
 
-  const authorizedProducts: Array<Product> =
+  const isAuthorizedAndAdminProducts: Array<Product> =
```

Comment:
don't use useMemo for isAuthorizedAndAdminProducts, you can use activeProduct for obtain Product already filtered by conditions, you can then compare them with the products onboarded for the institution by filtering those in userRole ADMIN and the authorized products